Go/Go.sublime-syntax

```diff
@@ -705,7 +721,7 @@ contexts:
 
   match-keyword-interface:
     - match: (?=\binterface\b)
-      scope: storage.type.keyword.interface.go
+      scope: keyword.declaration.interface.go
```

Comment:
Assigning a scope to a lookahead has no effect.

Reply:
Nice catch, thanks. Made a correction.